tools/s2n-events/src/generate_config.rs

```diff
@@ -143,10 +151,23 @@ impl GenerateConfig {
         }
     }
 
-    pub fn supervisor(&self) -> TokenStream {
+    fn supervisor_supported(&self) -> bool {
+        // The supervisor feature will require additional effort to expose a C API for. For now,
```

Comment:
Can you remind me what the supervisor does and why it's not necessary for the C APIs?